src/pudl/extract/epacems.py

```diff
@@ -100,7 +100,7 @@
 API_DTYPE_DICT = {
     "State": pd.CategoricalDtype(),
     "Facility Name": pd.StringDtype(),  # Not reading from CSV
-    "Facility ID": pd.Int16Dtype(),  # unique facility id for internal EPA database management (ORIS code)
+    "Facility ID": pd.Int32Dtype(),  # unique facility id for internal EPA database management (ORIS code)
```

Comment:
16 bit integers aren't actually large enough to hold these IDs, so some of them were wrapping around and becoming negative numbers.